### PR TITLE
Postgres Development Migration and testing

### PR DESCRIPTION
**Pull request by mmcphillips**
This pull request implements a change from the native mysql database to psql.
This pull request also adds metrics for testing and all required scripts for seeding, inserting, and csv writing.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,12 +4,21 @@
   "description": "Module to display featured film on Moodu",
   "main": "index.js",
   "scripts": {
+    "csvwrite": "--max_old_space_size=4096 server/csvwritepsql.js",
```
Comment (matt-leyrat) [inline package.json:7]: Did you give this a try with different numbers for max_old_space_size? I'm curious how much of a difference it makes.

File: server/csvinsertpsql.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+const dropTableQuery = `
+DROP TABLE IF EXISTS psqlFeatures;
+`;
+
+const copyTableQuery = `
+\copy psqlFeatures ( title,category_1,category_2,release_date,mpaa_rating,length,star_rating,star_rating_count,rt_rating,description,hd_rent,sd_rent,hs_cost,sd_cost,movie_shot_url,movie_cover_url ) FROM '${process.env.CSVD}' WITH DELIMITER ',' CSV HEADER;
+`;
+
+pool.query(dropTableQuery)
```
Comment (matt-leyrat) [inline server/csvinsertpsql.js:42]: super clear code 💯

File: server/helperFunctions.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+const { LoremIpsum } = require('lorem-ipsum');
+const { movies } = require('./movieList.js');
+
+const bgPath = 'https://featurefilm.s3-us-west-2.amazonaws.com/bg/wudu-background-';
+const featurePath = 'https://featurefilm.s3-us-west-2.amazonaws.com/f/wudu-feature-';
+const tagsArray = ['comedy', 'horror', 'drama', 'holiday special', 'action', 'adventure', 'scifi', 'fantasy', 'thriller', 'animated'];
+const mpaaRatings = ['G', 'PG', 'PG-13', 'R', 'NC-17', 'Unrated'];
+const movieNamesArray = movies.split('\n');
+const getRandomNumberTo100 = function () {
```
Comment (matt-leyrat) [inline server/helperFunctions.js:16]: You could combine all of these random number generators into a function that takes the max as a parameter

`getRandomNumber = (max) => { return Math.floor(Math.random() * max) + 1 }`

File: server/movieList.js [hunk 4]
```diff
@@ -1,103 +1,105 @@
...
-  Singin In The Rain
-  2001: A Space Odyssey
-  Toy Story
-  3 Idiots
-  The Sting
-  Toy Story 3
-  Inglourious Basterds
-  The Bicycle Thief
-  The Kid`,
+  movies:
```
Comment (matt-leyrat) [inline server/movieList.js:3]: I'm curious why this file is just exporting 100 lines of strings?

File: server/mysqlSeed.js [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+      values = [];
+    }
+  }
+}
+
+connection.connectAsync()
+  .then(() => db.queryAsync(`DROP DATABASE IF EXISTS ${process.env.DB_database}`))
+  .then(() => db.queryAsync(`CREATE DATABASE ${process.env.DB_database}`))
+  .then(() => db.queryAsync(`USE ${process.env.DB_database}`))
+  .then(() => db.queryAsync(`
```
Comment (matt-leyrat) [inline server/mysqlSeed.js:74]: Storing this query in a variable would make this code easier on the eyes 👁

File: tests/k6PostTest.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import http from 'k6/http';
+import { check, sleep } from 'k6';
+
+export const options = {
+  stages: [
```
Comment (matt-leyrat) [inline tests/k6PostTest.js:5]: Very nice, this seems to be the right way to use stages in k6.

**Review by matt-leyrat — APPROVED** [review]
Looks ready for a merge!

File: package.json (hunk 1 above)
Comment (mmcphillips) [reply package.json:7]: This didn't help me. I did try it but The solution for me was to change the max allocation by vscode instead

File: server/movieList.js (hunk 4 above)
Comment (mmcphillips) [reply server/movieList.js:3]: Its used as the grounds for randomly generated movie names for the titles

**Pull request closed without merging**